tests/unit/cluster/cluster-multiple-meets.tcl

```diff
@@ -0,0 +1,83 @@
+# make sure the test infra won't use SELECT
+set old_singledb $::singledb
+set ::singledb 1
+
+tags {tls:skip external:skip cluster} {
+    set base_conf [list cluster-enabled yes]
+    start_multiple_servers 2 [list overrides $base_conf] {
+        test "Cluster nodes are reachable" {
```

Comment:
Generalize this into a helper function?

unit/cluster/base.tcl has a similar test https://github.com/valkey-io/valkey/blob/unstable/tests/unit/cluster/base.tcl#L12